frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.ts

```diff
@@ -91,7 +92,8 @@ export class ChallengesettingsComponent implements OnInit, OnDestroy {
     private globalService: GlobalService,
     private apiService: ApiService,
     private endpointsService: EndpointsService,
-    private router: Router
+    private router: Router,
+    private logger: NGXLogger
```

Comment:
@ShauryaAg why are we adding a logger if we are not using it?

Reply:
This is also the result of my previous PR. Extremely sorry about this.